app/src/main/java/com/kylecombes/micrepeater/Information.java

```diff
@@ -0,0 +1,39 @@
+package com.kylecombes.micrepeater;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.text.method.LinkMovementMethod;
+import android.util.DisplayMetrics;
+import android.view.View;
+import android.widget.ImageButton;
+import android.widget.TextView;
+
+public class Information extends Activity {

```

Comment:
Change the class name `Information` to `InformationActivity`. You can do that by moving your cursor to the word Information, using `Shift + F6` (on Windows) to rename it. Android Studio will automatically find all usages and rename it for you.